compiler/circle2circle-dredd-recipe-test/test.lst

```diff
@@ -155,3 +155,4 @@ Add(Inf_Neg_000 PASS)
 Add(Inf_Pad_000 PASS)
 Add(Inf_StridedSlice_000 PASS)
 Add(Inf_StridedSlice_001 PASS)
+Add(Squeeze_dynamic_000 PASS)
```

Comment:
1/ plz follow naming convention
2/ add `test.rule` like others

Reply:
Q) what do you want to test with this model?

Reply:
> 1/ plz follow naming convention
done
> 2/ add test.rule like others
done

> Q) what do you want to test with this model?
I want to test scenario when squeezed dimension is unknown

Reply:
@seanshpark If you find a moment please take a look ;)

Reply:
changes in this PR looks OK. plz update related PRs.

Reply:
The related PRs are already updated

Reply:
> The related PRs are already updated

plz double check